subprojects/file-watching/src/main/java/org/gradle/internal/watch/registry/impl/AbstractFileWatcherUpdater.java

```diff
@@ -139,7 +142,7 @@ private void update(SnapshotHierarchy root) {
             });
     }
 
-    protected abstract void updateWatchesOnChangedWatchedFiles(FileHierarchySet watchedFiles);
+    protected abstract void updateWatchesOnChangedWatchedFiles(FileHierarchySet newWatchedFiles);
```

Comment:
Nit: revert rename?

Reply:
I prefer the new parameter name, given that it clarifies which watched files you receive and that HierarchicalFileWatcherUpdater already used `newWatchedFiles` as parameter name before this change.

Reply:
Good point.